examples/webgpu_mrt.html

```diff
@@ -106,11 +106,11 @@
 					const output = scenePass.getTextureNode( 'output' ); // output name is optional here
 					const normal = scenePass.getTextureNode( 'normal' );
 					const diffuse = scenePass.getTextureNode( 'diffuse' );
-					const emissive = scenePass.getTextureNode( 'emissive' );
+					const velocity = scenePass.getTextureNode( 'velocity' );
 
 					const out = mix( output.renderOutput(), output, step( 0.2, viewportTopLeft.x ) );
 					const nor = mix( out, normal, step( 0.4, viewportTopLeft.x ) );
-					const emi = mix( nor, emissive, step( 0.6, viewportTopLeft.x ) );
+					const emi = mix( nor, velocity, step( 0.6, viewportTopLeft.x ) );
```

Comment:
I hope it's okay to replace `emissive` here. It did it since we already use `emissive` in one of the bloom demos.

Reply:
> Shouldn't the velocity texture change when as camera is moved?

Um, I think it does. The effect is more visible if you move the camera quickly.

Reply:
> What coordinate frame is the velocity relative to?

It is computed in NDC space, no?

Reply:
Consider enabling damping. Be sure to call `controls.update()`.

I am surprised the velocity changes are not clearly visible....

Reply:
> Consider enabling damping. Be sure to call controls.update().

Done!